### DISCUSSION TITLE
QC calculation on 10x multiomic

### DISCUSSION THREAD

**Discussion by beginner984** [stuart-lab/signac#881] [Q&A] [answered]
Hello

In 10x multiomic website there are some explanation of metadata

https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/output/per_barcode_metrics

For joint RNA and ATAC analysis, I want to calculate these two but I don't know from which part I should get `blacklist_region_fragments` and `passed_filters`

`> pbmc$pct_reads_in_peaks <- pbmc$atac_peak_region_fragments/  pbmc$passed_filters * 100`

`> pbmc$blacklist_ratio <- pbmc$blacklist_region_fragments / pbmc$atac_peak_region_fragments`

Thanks for any help

**Comment by timoast** [comment, marked as answer]
1. The `passed_filters` metric in scATAC is equivalent to `atac_fragments` in multiome
2. As stated several times previously (https://github.com/timoast/signac/discussions/878), an example showing how to compute the blacklist ratio is shown at the end of this Signac vignette (under the dropdown section at the end): https://satijalab.org/signac/articles/pbmc_vignette.html
